doc/rst/source/gallery/ex53.rst

```diff
@@ -0,0 +1,42 @@
+.. _example_53:
+
+(53) Building subplots and using per-country shading
+----------------------------------------------------
+
+This example illustrates two separate effects:
+
+#. How to automate the plotting of a large set of subplot panels
+   via a simple loop that picks up panel-specific parameters from
+   a configuration file.
+#. How to highlight a country using a DEM with darker ambient shading
+   outside the country boundaries from DCW.
+
+We use the 6 arc-minute global relief file for these plots and show
+topographic relief in an equal-area block of 3000 by 3000 km centered
+on one of the dozen most populated cities in the world. Apart from
+the metadata for each city, all subplot panels are identical, thus
+making a simple loop over the metadata preferable to writing 12
+near-identical subplot sections.  **Note**: For optimal resolution
+you should switch to the 2x2 arc-minute relief data instead.
+
+We use a small table of metadata for the 12 cities obtained from the
+full listing at https://simplemaps.com/data/world-cities and created
```

Comment:
```suggestion
full listing at https://simplemaps.com/data/world-cities and create
```

Reply:
Or another option is "https://simplemaps.com/data/world-cities, which was used to create the metadata file with"... Not sure what is correct here, but the single past tense seemed off.

Reply:
Yes, a bit odd. Now rewritten.